src/pages/management/changelog/index.mdx

```diff
@@ -23,6 +23,10 @@ The developer preview version provides a preview of the upcoming changes to the
 
 ## [v3.5] - Developer preview
 
+### Other
+
+  - There's a new method, [**Check Product Limits for Plan**](/management/configuration-api/v3.5/#get-product-limits-for-plan).
```

Comment:
The naming is inconsistent across files - Check/Get Product Limits for Plan

Reply:
fixed here https://github.com/livechat/livechat-public-docs/pull/1099/commits/4d6c871c209b48a16f586f6bf246ba617f9017eb